src/video/shaders.cpp

```diff
@@ -257,7 +257,7 @@ static GLint FrameCount;
 static GLint OutputSize;
 static GLint TextureSize;
 static GLint InputSize;
-static GLint VertexCoord;
+// static GLint VertexCoord;
```

Comment:
```suggestion
```

Reply:
fixed

Reply:
Seems you forget to push the changes ;-)

Reply:
The commit is on the branch I created the PR from. But github PR does not show the added changes.
Branch with 3 commits: https://github.com/zzam/stratagus/tree/fix-warnings